### ISSUE TITLE
复现时遇到的一些问题

### ISSUE THREAD

**Issue by tosvqo** [ZPDu/Redesigning-Multi-Scale-Neural-Network-for-Crowd-Counting#4]
作者您好，感谢您开源的代码以及对论文的贡献，在复现时遇到了一些问题还希望解答：
1、# density loss
            loss += (2**(int(j / 3))) * mseloss(pred_maps[j].squeeze(), gt_map) 
这里pred_maps应该是有4个维度，类似1,1,H,W但gt_map输出的形状是1,H,W，如果squueze()会导致pred_maps->H,W。如果这里不修改运行程序会报错。
2、if (i + 1) % cfg_data.PRINT_FREQ == 0:
        loss = mseloss(pred_maps[0].squeeze(), gt_map)
这里也是一样的问题。
3、在def validate中，您使用了pred_map = net(img)[0]这一句，会得到一个5维形状的参数，然而在计算loss = mseloss(pred_map, gt_map)这里时固然也会报错。还是维度不匹配的报错问题。
4、最后我自己通过unsqueeze和squeeze修改维度至匹配后，复现出来的MAE效果只有60 70 80 90 甚至100，远远达不到您所给的权重文件的效果。
希望作者百忙之中可以解答一下。

**Comment by tosvqo** [comment]
还有就是训练文件中貌似您将relloss注释掉了，我加上之后效果更差，梯度甚至直接消失，预测为0
